### ISSUE TITLE
Solver.prototxt for sintel and kitti pretrained models.

### ISSUE THREAD

**Issue by TusharNimbhorkar** [lmb-freiburg/flownet2#124]
Hey

I was wondering what `solver.prototxt` files you used for `sintel` and `kitti` models.

And are these models got only trained on this two dataset.

**Comment by nikolausmayer** [comment]
Our published models for Sintel and KITTI are pretrained on [FlyingThings3D](https://lmb.informatik.uni-freiburg.de/resources/datasets/SceneFlowDatasets.en.html), then finetuned on the respective dataset’s training set.

The `solver.prototxt` file is exactly the same as the one for pretraining—with the exception of the crop parameters. Sintel and KITTI have different image sizes and aspect ratios, and the crop size for FlyingThings3D would not work.

**Comment by TusharNimbhorkar** [comment]
Thanks for the reply.

One more thing. Do you have anything to check epe while training?

**Comment by nikolausmayer** [comment]
We just print loss values from the loss layers, nothing more fancy

**Comment by TusharNimbhorkar** [comment]
Most of the statistics in the Flownet2 paper are given in EPE. Do you calculate that separately?

**Comment by TusharNimbhorkar** [comment]
Ah okay, so the Loss which is shown during training is basically EPE.

**Comment by nikolausmayer** [comment]
Yes

**Comment by TusharNimbhorkar** [comment]
Thanks again closing the issue.
